src/main/lrsql/admin/interceptors/account.clj

```diff
@@ -131,7 +131,9 @@
              {:keys [account-id]} ::data}
             ctx
             {:keys [result]}
-            (adp/-delete-account lrs account-id)]
+            (adp/-delete-account
+             lrs account-id
+             (some? (:lrsql.admin.interceptors.oidc/admin-env ctx)))]
```

Comment:
Maybe extract this value out into its own let-binding

Reply:
https://github.com/yetanalytics/lrsql/pull/294/commits/9bcd9cee9ca161af866cc094c559b9cf0dcc6eba